index.html

```diff
@@ -1,27 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Simple TODO List</title>
-    <link rel="stylesheet" href="index.css">
-</head>
-<body>
-    <h1>Simple TODO list</h1>
-    <div id="todo-list">
-      <!-- Your code goes here (Part 1) -->
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <div id="todo-app">
+      <!-- Todo list heading -->
+      <h1>Simple TO-DO List</h1>
+      <ul id="todo-list">
+        <!--  Todo items will  be shown here -->
+      </ul>
+      <!-- Input fields for todo -->
+      <label>Title</label>
+      <input type="text" id="new-todo-title" />
+      <label>Description</label>
+      <input type="text" id="new-todo-description" />
+      <button id="add-todo">Add to list</button>
     </div>
+
     <script>
-      // Your JS goes here (Part 2)
-      
-      // This data array contains the initial 3 todo items
-      // When you are done with writing the code for processing the list
-      // Remove this and store the data in localstorage
       var data = [
-        { id: 1, title: 'Learn JavaScript', completed: true },
-        { id: 2, title: 'Learn React', completed: false },
-        { id: 3, title: 'Build a simple project', completed: false }
-      ]
+        { id: 1, title: "Learn JavaScript", completed: true },
+        { id: 2, title: "Learn React", completed: false },
+        { id: 3, title: "Build a simple project", completed: false },
+      ];
+      // Function to update todo display
+      function updateDisplay() {
+        var todoList = document.getElementById("todo-list");
+        todoList.innerHTML = "";
+        data.forEach(function (item) {
+          var li = document.createElement("li");
+          li.className = "todo-item" + (item.completed ? " completed" : "");
+          li.innerHTML = `
```

Comment:
Unfortunately, inner HTML is a serious security risk. Not quite in this case, but it can be if input is passed to it: https://www.tutorialspoint.com/disadvantages-of-using-innerhtml-in-javascript#:~:text=InnerHTML%20does%20not%20provide%20proper,which%20can%20cause%20unexpected%20issues. 

It's better to use createChild/appendChild. 

-1